### PR TITLE
app-editors/neovim: add 0.7.0; drop 0.4.4-r101

### PR DESCRIPTION
**Pull request by Kangie**
All looks good; trivial version bump.

Dropped the oldest version - there are two other versions with the same arches unmasked that are newer.

### CODE REVIEW

File: app-editors/neovim/neovim-0.7.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (thesamesam) [inline app-editors/neovim/neovim-0.7.0.ebuild]: 8 work?

File: app-editors/neovim/neovim-0.7.0.ebuild [hunk 2]
```diff
@@ -15,52 +15,56 @@ if [[ ${PV} == 9999 ]]; then
...
 BDEPEND="${LUA_DEPS}
-	dev-util/gperf
+	>=dev-util/gperf-3.1
 	virtual/libiconv
 	virtual/libintl
 	virtual/pkgconfig
 "
-# TODO: add tests, dev-lua/busted has now got luajit support.
-# bug #584694
+# Check https://github.com/neovim/neovim/blob/master/third-party/CMakeLists.txt for
```
Comment (thesamesam) [inline app-editors/neovim/neovim-0.7.0.ebuild:39]: Did you remember to do these bits? pkgdiff -b shows some changes. I'll sort it though, minor stuff

**Comment by thesamesam** [comment]
Merged! But I squashed the fixup bits so lost the Closes tag.

File: app-editors/neovim/neovim-0.7.0.ebuild (hunk 1 above)
Comment (Kangie) [reply app-editors/neovim/neovim-0.7.0.ebuild]: I thought I fixed that as part of the bump! I'll raise a new PR to bump the EAPI this evening if it all tests OK.

**Pull request closed without merging**